app/src/org/commcare/provider/InstanceProvider.java

```diff
@@ -520,7 +520,8 @@ private void linkToSessionFormRecord(Uri instanceUri) {
             return;
         }
 
-        Logger.log(AndroidLogger.TYPE_FORM_ENTRY, "Form Entry Completed");
+        Logger.log(AndroidLogger.TYPE_FORM_ENTRY,
+                String.format("Form Entry Completed for record with id %s", current.getInstanceID()));
```

Comment:
Looks like what we want: 

<img width="1187" alt="screen shot 2017-02-22 at 11 55 44 am" src="https://cloud.githubusercontent.com/assets/2293064/23206372/0f8db01c-f8f6-11e6-87b0-9f2dba1e6070.png">


02-22 11:53:52.346 14502-14927/org.commcare.dalvik W/System.err: logger> form-entry: Form Entry Completed for record with id 9b2efabe-4ad3-47b2-8835-5b76adbdfd36
02-22 11:53:57.254 14502-14729/org.commcare.dalvik W/System.err: logger> form-submission: Successfully submitted form with id 9b2efabe-4ad3-47b2-8835-5b76adbdfd36